packages/clay-card/src/Body.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import React from 'react';
+
+import classNames from 'classnames';
+
+import Context from './Context';
+
+export interface CardBodyProps extends React.HTMLAttributes<HTMLDivElement> {
+	children:
+		| React.ReactElement<HTMLElement>[]
+		| React.ReactElement<HTMLElement>;
+}
+
+export const Body: React.FunctionComponent<CardBodyProps> = ({
+	children,
+	className,
+	...otherProps
+}) => {
+	const {horizontal, interactive} = React.useContext(Context);
+
+	const isCardType = {
```

Comment:
create this object here, is it necessary? since we are not doing anything special here, we can call only `horizontal` and `interactive`.

Reply:
Solved.